wvd-templates/DSC/Configuration.ps1

```diff
@@ -37,122 +38,53 @@ configuration FirstSessionHost
         [string]$EnablePersistentDesktop = "False",
 
         [Parameter(Mandatory = $true)]
-        [string]$DefaultDesktopUsers
+        [string]$DefaultDesktopUsers,
+
+        [Parameter(mandatory = $false)]
+        [string]$RDPSModSource = 'attached'
     )
 
-    $rdshIsServer = $true
+    $ErrorActionPreference = 'Stop'
+
     $ScriptPath = [system.io.path]::GetDirectoryName($PSCommandPath)
+    . (Join-Path $ScriptPath "Functions.ps1")
 
-    $OSVersionInfo = Get-ItemProperty "HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion"
-    
-    if ($OSVersionInfo -ne $null)
-    {
-        if ($OSVersionInfo.InstallationType -ne $null)
-        {
-            $rdshIsServer=@{$true = $true; $false = $false}[$OSVersionInfo.InstallationType -eq "Server"]
-        }
-    }
+    $rdshIsServer = isRdshServer
 

```

Comment:
@ChristianMontoya Not sure where your comment is about this, so replying here:
I created a new function `isRdshServer` inside Functions.ps1